mteb/abstasks/TaskMetadata.py

```diff
@@ -214,6 +214,8 @@ def _check_language_code(code):
         This method checks that the language code (e.g. "eng-Latn") is valid.
         """
         lang, script = code.split("-")
+        if script == "Code":
+            return  # override for code
```

Comment:
```suggestion
        if script == "Code":
            if lang in PROGRAMMING_LANGS # you can create this list in the same file as where languages is loaded in
              return  # override for code
            else:
               raise ValueError(f"Programming language {lang} is not a valid programming language.")
```
I would validate the list of programming languages as well (to make sure we e.g. don't have "python" and "Python".